packages/alpine.rb

```diff
@@ -2,11 +2,11 @@
 
 class Alpine < Package
   description 'The continuation of the Alpine email client from University of Washington.'
-  homepage 'http://alpine.x10host.com/alpine'
+  homepage 'https://alpineapp.email/'
   version '2.25'
   license 'Apache-2.0'
   compatibility 'all'
-  source_url 'http://alpine.x10host.com/alpine/release/src/alpine-2.25.tar.xz'
+  source_url 'https://alpineapp.email/alpine/release/src/Old/alpine-2.25.tar.xz'
```

Comment:
Update to https://alpineapp.email/alpine/release/src/alpine-2.26.tar.xz?

Reply:
This will prove the new domain compiles correctly with the current package configuration.

Reply:
![image](https://github.com/chromebrew/chromebrew/assets/64618338/547d10e2-9b16-4127-835f-c0b5f8027dd4)
Its the same file with the same hash.

Reply:
The amount of effort to prove the code is the same could be better spent going ahead with the update but I digress...